Sources/MapboxNavigation/VisualInstruction.swift

```diff
@@ -85,10 +85,11 @@ extension VisualInstruction {
     /// Returns whether the `VisualInstruction`’s maneuver image should be flipped according to the driving side.
     public func shouldFlipImage(side: DrivingSide) -> Bool {
         switch maneuverType ?? .turn {
+        case _ where maneuverDirection == .uTurn:
+            return side == .right
```

Comment:
This make sense, but double-check that this has the expected results in:

* iPhone and CarPlay
* U-turns at ordinary intersections and roundabouts
* when using OSRM and Valhalla endpoints or profiles
* in right-hand-driving and left-hand-driving regions
* in left-to-right and right-to-left environments (based on the language on iPhone, based on the regional driving side in CarPlay)